lib/pagerduty_client.rb

```diff
@@ -59,7 +59,7 @@ def shifts_to_override(shifts_to_assign, assigned_shifts_this_schedule)
     end
   end
 
-  def pagerduty_shifts_to_override(shift_to_assign, existing_pagerduty_shifts)
+  def shifts_within_timespan(shift_to_assign, existing_pagerduty_shifts)
```

Comment:
TODO: split out into start/end datetime parameters, and `existing_pagerduty_shifts`.

And then fix up the method restructuring / renaming to the commits where they're first introduced.